### PR TITLE
MAINT: Reenable dask

### LINKED ISSUE #548: Fix Dask support
#547 disabled Dask in our CI tests, as recent changes in MNE-Python lead to a resource oversubscription. We need to take a closer look to figure out how to adjust the pipeline accordingly.

cc @agramfort @larsoner

### PR DESCRIPTION
**Pull request by larsoner**
### Before merging …

- [ ] Changelog has been updated (`docs/source/changes.md`)

Closes #548

TDD, first let's see the failure.

No need for changelog update I think because it wasn't updated when dask was disabled, either.

### CODE REVIEW

File: config.py [hunk 1]
```diff
@@ -3475,8 +3475,7 @@ def _load_data(cfg, bids_path):
         picks = get_channels_to_analyze(raw.info)
         raw.pick(picks)
 
-    if bids_path.subject != 'emptyroom':
-        _crop_data(cfg, raw=raw, subject=subject)
+    _crop_data(cfg, raw=raw, subject=subject)
```
Comment (larsoner) [inline config.py:3478]: No need for this conditional because `_crop_data` already has it

File: tests/configs/config_ds001971.py [hunk 2]
```diff
@@ -1,4 +1,4 @@
-"""Configuration file for the ds001971 dataset.
+"""Mobile brain body imaging (MoBI) gait adaptation experiment.
```
Comment (larsoner) [inline tests/configs/config_ds001971.py:1]: Looking at the website I noticed that these descriptions were not great, so I made them at least a little bit better.

File: tests/configs/config_ds000246.py [hunk 3]
```diff
@@ -1,12 +1,16 @@
...
+See https://openneuro.org/datasets/ds000246/versions/1.0.0 for more
+information.
 """
 
 study_name = 'ds000246'
 bids_root = '~/mne_data/ds000246'
 deriv_root = '~/mne_data/derivatives/mne-bids-pipeline/ds000246'
 
 runs = ['01']
+crop_runs = (0, 180)  # Reduce memory usage on CI system
```
Comment (larsoner) [inline tests/configs/config_ds000246.py:13]: We usually have to crop CTF data in our MNE examples. I'm hoping that cropping it a bit here will similarly help

**Comment by larsoner** [comment]
All green, ready for review/merge from my end

**Review by agramfort — APPROVED** [review]

**Comment by agramfort** [comment]
thx @larsoner

**Pull request merged**